llvm/test/Transforms/SROA/strict-align-offset-transfer-2.ll

```diff
@@ -0,0 +1,127 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --function-signature --scrub-attributes
+; RUN: opt -S -passes=sroa -o - < %s | FileCheck %s
+; RUN: sed -e 's/addrspace(200)/addrspace(0)/g' -e 's/-A200-P200-G200//g' -e 's/i64 32/i64 16/g' -e 's/\.p200i8/\.p0i8/g' %s | opt -S -passes=sroa -o - | FileCheck %s --check-prefix NOCHERI
```

Comment:
```suggestion
; RUN: sed -e 's/addrspace(200)/addrspace(0)/g' -e 's/-A200-P200-G200//g' -e 's/i64 32/i64 16/g' -e 's/\.p200i8/\.p0i8/g' %s \
; RUN:   | opt -S -passes=sroa -o - | FileCheck %s --check-prefix NOCHERI
```
is a bit easier on the eyes

Reply:
Maybe also use `@CAP_PAIR_SIZE@` as a template for both rather than replacing i64 32? Seems a bit brittle.

Reply:
Perhaps also just replace the 200's everywhere with `@CAP_ADDRSPACE@` or something? That way you don't need to substitute all the different ways it's used.

Reply:
Sounds good. I'll push an updated patch soon.